client/blocks/post-status/index.jsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+import { getNormalizedPost } from 'state/posts/selectors';
+
+function PostStatus( { translate, post } ) {
+	if ( ! post ) {
+		return null;
+	}
+
+	const { sticky, status } = post;
+	let text, classModifier, icon;
+	if ( sticky ) {
+		text = translate( 'sticky' );
+		classModifier = 'is-sticky';
+		icon = 'bookmark-outline';
+	} else if ( 'pending' === status ) {
+		text = translate( 'Pending review' );
+		classModifier = 'is-pending';
+	} else if ( 'future' === status ) {
+		text = translate( 'Scheduled' );
+		classModifier = 'is-scheduled';
+		icon = 'calendar';
+	} else if ( 'trash' === status ) {
+		text = translate( 'trashed' );
```

Comment:
>Should this be capitalized like the rest of the `text` assignments above?

Yeah, I think we should be consistent here with the translate strings, even though the styling will force them to the same appearance.